build.sbt

```diff
@@ -55,9 +87,17 @@ lazy val publishSettings = Def.settings(
       "Adrien Piquerez",
       "adrien.piquerez@gmail.com",
       url("https://github.com/adpi2")
+    ),
+    Developer(
```

Comment:
Do you wish to become a maintainer or more of a regular contributor?

Reply:
not sure - it is your call. :smile_cat:

Reply:
but I can certainly help as a maintainer